types/iso-url/index.d.ts

```diff
@@ -0,0 +1 @@
+export { URL, URLSearchParams }
```

Comment:
hmm, I thought TS did not liked exports of things that aren't defined locally.

Reply:
our config has the browser/dom stuff